x-pack/plugins/snapshot_restore/public/application/sections/home/snapshot_list/snapshot_details/tabs/snapshot_state.tsx

```diff
@@ -5,84 +5,66 @@
  * 2.0.
  */
 
-import React, { Fragment } from 'react';
+import React from 'react';
 
-import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiIconTip, EuiLoadingSpinner } from '@elastic/eui';
+import { EuiFlexGroup, EuiHealth, EuiIcon, EuiToolTip } from '@elastic/eui';
 
 import { SNAPSHOT_STATE } from '../../../../../constants';
 import { useServices } from '../../../../../app_context';
 
 interface Props {
   state: any;
+  tooltipIcon: boolean;
```

Comment:
Can we rename this prop to something more descriptive (e.g. `addStateTooltipIcon`)?

Reply:
changed to `displayTooltipIcon`